scene/resources/visual_shader.h

```diff
@@ -350,6 +350,10 @@ class VisualShaderNodeInput : public VisualShaderNode {
 
 	String input_name = "[None]";
 
+public:
+	void _set_shader_type(VisualShader::Type p_shader_type);
+	void _set_shader_mode(Shader::Mode p_shader_mode);
```

Comment:
Public methods should probably not start with `_`.

Reply:
Fixed.